### PR TITLE
Update typed error to the latest version

### PR DESCRIPTION
**Pull request by LucianBuzzo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker: # use the docker executor type; machine and macos executors are also supported
-      - image: circleci/node:9.9.0 # the primary container, where your job's commands are run
+      - image: circleci/node:10.18.1 # the primary container, where your job's commands are run
```
Comment (Page-) [inline .circleci/config.yml:6]: It's probably worth using 12.x since that's what it'll end up being run on

**Comment by karaxuna** [comment]
@LucianBuzzo the reason why it was failing was that TypedError module used es6 classes, while code in our lib was targeted to es5 so something like this happened:

```javascript
TypedError.call(this, ...) // <- this throws an error, because TypedError should be called with new or extended by class
```

**Comment by LucianBuzzo** [comment]
@balena-ci I self-certify!

**Comment by LucianBuzzo** [comment]
@balena-ci I self-certify!

**Comment by LucianBuzzo** [comment]
@balena-ci I self certify!

**Review by ljewalsh — APPROVED** [review]

**Pull request merged**